calender-venv/Scripts/Activate.ps1

```diff
@@ -0,0 +1,51 @@
+function global:deactivate ([switch]$NonDestructive) {
```

Comment:
Prefer to not upload your virtual environment files.
Use .gitignore to ignore them, and `git rm` to remove them from the repo.